dbt/adapters/athena/column.py

```diff
@@ -28,6 +29,9 @@ def is_binary(self) -> bool:
     def is_timestamp(self) -> bool:
         return self.dtype.lower() in {"timestamp"}
 
+    def is_array(self) -> bool:
```

Comment:
I've tried my best to follow the existing interface for parsing types, which appears to come from way that the underlying `Column` code is written.